screens/HomeScreen.tsx

```diff
@@ -41,6 +46,9 @@ export default function HomeScreen({navigation}: NativeStackHeaderProps) {
       <Button onPress={() => navigation.navigate('MuscleGroup')}>
         <ButtonText>Generate</ButtonText>
       </Button>
+      <TouchableOpacity onPress={clearOnboarding}>
+        <Text>Clear onboarding</Text>
+      </TouchableOpacity>
```

Comment:
There is to button to clear on-boarding? We should clear it when users finish on-boarding automatically